e2e/scheduler_test.go

```diff
@@ -17,6 +17,7 @@ var capacityPodTemplateYAML string
 func testScheduler() {
 	var cc CleanupContext
 	BeforeEach(func() {
+		skipIfStorageCapacity()
```

Comment:
I think we have to confirm pods are scheduled using capacity tracking in an end-to-end manner.
Or just adding a unit test for GetCapacity is sufficient.
I want your thought.

Reply:
> I think we have to confirm pods are scheduled using capacity tracking in an end-to-end manner.
> Or just adding a unit test for GetCapacity is sufficient because scheduling must be tested by kubernetes.
> May I ask you your thought?

Um...I think storage capacity scheduling tests are written sufficiently by Kubernetes like below:
- https://github.com/kubernetes/kubernetes/blob/v1.21.1/pkg/controller/volume/scheduling/scheduler_binder_test.go#L2202-L2354
- https://github.com/kubernetes/kubernetes/blob/v1.21.1/test/e2e/storage/testsuites/capacity.go

So I think it's enough to add tests for setting the CSIStorageCapacity resource by the csi-provisioner and topolvm GetCapacity.
What do you think?

Reply:
I noticed that can't write test cases for storage capacity tracking because k8s.io/api package version is v0.20.2 currently...
It is necessary to upgrade k8s.io/api and controller-runtime to use CSIStorageCapacity resource in e2e testing.

Reply:
I agreed to add a test for the CSIStorageCapacity resource.